### PR TITLE
notebook 7.2.0

### PR DESCRIPTION
**Pull request by bollwyvl**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [ ] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by bollwyvl** [comment]
@conda-forge/notebook any thoughts?

File: recipe/meta.yaml [hunk 1]
```diff
@@ -53,7 +53,6 @@ about:
   license_file:
     - LICENSE
     - notebook/labextension/static/third-party-licenses.json
-    - notebook/static/third-party-licenses.json
```
Comment (jtpio) [inline recipe/meta.yaml:56]: Normally this file should still exist in the upstream tarball. But no strong opinion on whether we should keep it or not in the recipe.

Comment (bollwyvl) [reply recipe/meta.yaml:56]: ah, let's try

Comment (bollwyvl) [reply recipe/meta.yaml]: [24e5ba6](https://github.com/conda-forge/notebook-feedstock/pull/155/commits/24e5ba669cd979ebe7fcd083d10e3635978d0010)

**Pull request merged**